srcpkgs/openfst/template

```diff
@@ -0,0 +1,59 @@
+# Template file for 'openfst'
+pkgname=openfst
+version=1.8.2
+revision=1
+build_style=gnu-configure
+configure_args="
+	--disable-bin
+	--disable-dependency-tracking
+	--enable-compact-fsts
+	--enable-compress
+	--enable-const-fsts
+	--enable-far
+	--enable-linear-fsts
+	--enable-lookahead-fsts
+	--enable-mpdt
+	--enable-ngram-fsts
+	--enable-pdt
+	--enable-shared
+	--enable-static
+	--with-pic
+"
+hostmakedepends="autoconf automake libtool texinfo"
+short_desc="Finite-state transducers library"
+maintainer="John Gebbie <me@johngebbie.com>"
+license="Apache-2.0"
+homepage="https://www.openfst.org"
+distfiles="https://www.openfst.org/twiki/pub/FST/FstDownload/openfst-$version.tar.gz"
+checksum=de987bf3624721c5d5ba321af95751898e4f4bb41c8a36e2d64f0627656d8b42
+
+post_extract() {
+	# Don't run test during build.
+	vsed -i configure.ac -e '152,184 d'
+	# Remove -msse check confused by cross compiling.
+	vsed -i src/include/fst/float-weight.h -e '120 d'
+}
```

Comment:
specifying which lines should be deleted is very fragile, I think a patch would be better.

Reply:
Thank you so much, made me very happy seeing you'd reviewed!

I added a patch to allow cross compiling, but a test about float equality fails when natively compiling i686, so I have limited the template to x86_64* and aarch64* with a comment to the issue I've opened, until it is resolved.

I see they're doing the same on alpine: https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/41097